ex5/dist/style.css

```diff
@@ -105,13 +100,52 @@ body {
 }
 
 .list-item-delete-button {
-  margin-left: auto;
-  padding: 8px;
   cursor: pointer;
-  border-radius: 8px;
+  float: right;
+  height: auto;
+  width: auto;
 }
 
 .list-item-delete-button:hover {
   background-color: #f0f0f5;
   transition: background-color 0.2s ease-in-out;
 }
+
+.NoTasksImg-visible {
+  padding: 30px;
+  display: block;
+  margin: auto;
+  height: 1000%;
+  overflow: auto;
+}
+
+.noTasks-img-hidden {
+  display: none;
+  margin: auto;
+  width: 50%;
+}
+
+.pendingTasksCounter-visible {
```

Comment:
be consistent with naming - all CSS classes should be of format kebab-case
this class should be named pending-tasks-counter-visible